tests/create_test.go

```diff
@@ -366,7 +366,7 @@ func TestCreateInvalidSlice(t *testing.T) {
 		nil,
 	}
 
-	if err := DB.Create(&users).Error; !errors.Is(err, gorm.ErrInvalidData) {
+	if err := DB.Create(&users).Error; !errors.Is(err, gorm.ErrInvalidValue) {
```

Comment:
This test error change form `ErrInvalidData` to `ErrInvalidValue`, seems not be a breaking change.